2020/day3.clj

```diff
@@ -1,89 +0,0 @@
-(ns advent-of-code.2020.day3
-  (:require [clojure.string :as string]))
-
-(defn file->seq
-  [filename]
-  (->> (slurp filename)
-       (string/split-lines)))
-
-(defn next-location
-  [loc [right down]]
-  [(+ (first loc) right)
-   (+ (second loc) down)])
-
-(defn locations-s
-  [height [right down]]
-  (take-while #(< (second %) height)
-              (map #(next-location [0 0] [(* right %) (* down %)])
```

Comment:
Use full anonymous function form for any non-trivial fn definitions

Reply:
This also applies to other # fn forms... replace with full function definition

Reply:
> Use full anonymous function form for any non-trivial fn definitions

rewrote in second PR.